UnitySDK/Assets/ML-Agents/Examples/SharedAssets/Scripts/SettingsOverrides.cs

```diff
@@ -0,0 +1,52 @@
+using UnityEngine;
+using MLAgents;
+
+public class SettingsOverrides : MonoBehaviour
+{
+    // Original values
+    Vector3 m_OriginalGravity;
+    float m_OriginalFixedDeltaTime;
+    float m_OriginalMaximumDeltaTime;
+    int m_OriginalSolverIterations;
+    int m_OriginalSolverVelocityIterations;
+
+    public float gravityMultiplier = 1.0f;
+    public float fixedDeltaTime = .02f;
```

Comment:
@surfnerd Do you know if it's possible to read these from the project settings? I tried defaulting it to `Time.fixedDeltaTime` but that's apparently not allowed.

Reply:
> UnityException: get_fixedDeltaTime is not allowed to be called from a MonoBehaviour constructor (or instance field initializer), call it in Awake or Start instead. Called from MonoBehaviour 'SettingsOverrides' on game object 'Ball3DAcademy'.

Reply:
I'm not sure of the nuances in object loading, but the settings are loaded from assets, and perhaps when a Monobehaviour is instantiated vs. when Awake/Start is called, there is a possibility that these settings are loaded yet.

Reply:
hunch seems to be confirmed [here](https://blogs.unity3d.com/2016/06/06/serialization-monobehaviour-constructors-and-unity-5-4/)